.travis.yml

```diff
@@ -97,6 +98,7 @@ jobs:
         - echo "check if luigi-client.d.ts exists" && cat $TRAVIS_BUILD_DIR/test/e2e-test-application/node_modules/@luigi-project/client/luigi-client.d.ts | grep -C 3 "withoutSync" || travis_terminate 1
         - lerna run build --ignore docs || travis_terminate 1
         - npm test --prefix core || travis_terminate 1
+        - npm test --prefix client-frameworks-support/client-support-angular/
```

Comment:
I see what you are attempting. I would not expect this line here. Instead you need to add a new Travis **stage/substage** , like [this](https://github.com/SAP/luigi/blob/master/.travis.yml#L65),  [this](https://github.com/SAP/luigi/blob/master/.travis.yml#L106), or [this](https://github.com/SAP/luigi/blob/master/.travis.yml#L119) .

If you are within a stage, you will be using the chrome headless by default I presume, as other similar tests don't have any special flags.